components/webui/server/src/app.ts

```
@@ -1,25 +1,106 @@
-    await fastify.register(staticRoutes);
-};
-
-export default FastifyV1App;
+// eslint-disable-next-line max-lines-per-function
+export default async function serviceApp (
+    fastify: FastifyInstance,
+    opts: FastifyPluginOptions
+) {
+    // Option only serves testing purpose. It's used in testing to expose all decorators to the
+    // test app. Some decorators may not be exposed in production.
+    delete opts.skipOverride;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)